stdlib/private/SwiftReflectionTest/SwiftReflectionTest.swift

```diff
@@ -470,8 +476,10 @@ public func reflect(function: @escaping (Int, String, AnyObject?) -> Void) {
       capacity: MemoryLayout<ThickFunction3>.size)
   fn.initialize(to: ThickFunction3(function: function))
 
-  let parts = unsafeBitCast(fn, to: UnsafePointer<ThickFunctionParts>.self)
-  let contextPointer = unsafeBitCast(parts.pointee.context, to: UInt.self)
+  let contextPointer = fn.withMemoryRebound(
+    to: ThickFunctionParts.self, capacity: 1) {
+      UInt(bitPattern: $0.pointee.context)
+  }
```

Comment:
Since the function is written to memory only to extract its parts, It might be a bit cleaner to rewrite this code using `RawPointer`s throughout to keep the memory untyped:

```
let fn = UnsafeMutableRawPointer.allocate(bytes: MemoryLayout<ThickFunction3>.size,
                                          alignedTo: MemoryLayout<ThickFunction3>.alignment)
fn.initializeMemory(as: ThickFunction3.self, to: ThickFunction3(function: function))
let contextPointer = fn.load(fromByteOffset: 0, as: UInt.self)
let codePointer = fn.load(fromByteOffset: MemoryLayout<UInt>.size, as: UInt.self)
```